### PR TITLE
Initial Gecos LightDM configuration files

### PR DESCRIPTION
**Pull request by joroca**

### CODE REVIEW

File: gcs/depends [hunk 1]
```diff
@@ -0,0 +1,3 @@
+lightdm-gtk-greeter
+gecosws-artwork
```
Comment (alfem) [inline gcs/depends:2]: Why does this package depend on gecosws-artwork?

File: gcs/conffiles_skel/etc/lightdm/lightdm-gtk-greeter.conf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[greeter]
+background = /usr/share/lightdm-gtk-greeter/themes/gecos/bg.jpg
+theme-name = Gecos
```
Comment (alfem) [inline gcs/conffiles_skel/etc/lightdm/lightdm-gtk-greeter.conf:4]: Please, use GECOS always in uppercase

**Review by alfem — CHANGES_REQUESTED** [review]

File: gcs/conffiles_skel/etc/lightdm/lightdm-gtk-greeter.conf (hunk 2 above)
Comment (joroca) [reply gcs/conffiles_skel/etc/lightdm/lightdm-gtk-greeter.conf:4]: theme-name option is case-sensitive

File: gcs/depends (hunk 1 above)
Comment (joroca) [reply gcs/depends:2]: Because gtk greteer depends on Gecos theme and it belongs to gecosws-artwork package.

Comment (alfem) [reply gcs/depends:2]: Ok

**Pull request merged**